README.md

```diff
@@ -75,12 +76,12 @@ input_pairs = [
     }
 ]
 
-affinities = []
-poses = []
+ligand_affinities = []
+poses_predictions = []
```

Comment:
I have a question about terminology- would it make sense to name `pose_predictions` as `pose_affinities`? It would be more consistent with `ligand_affinities`- I'm just not sure if one of these terminologies is "more correct" in this case? If they're equivalent- I would suggest renaming this variable into `pose_affinities`, the variable in `InferenceResults` dataclass and everywhere else in this PR.

Reply:
the reason its `pose_predictions` and not `pose_affinities` is that this CSV has the predictions for both `affinity` and `confidence` for each pose. Unlike the `ligand_affinitites` CSV which has the predictions only for `affinity` for each ligand.

Reply:
Ok, got it, makes sense then!